examples/contrib/modbus_mapper.py

```diff
@@ -16,36 +16,84 @@
 simplified in the final api; it is just an example of the
 requested functionality)::
 
-    from modbus_mapper import csv_mapping_parser
-    from modbus_mapper import mapping_decoder
-    from pymodbus.client import ModbusTcpClient
-    from pymodbus.payload import BinaryModbusDecoder
+    CSV:
+    address,type,size,name,function
+    1,int16,2,Comm. count PLC,hr
+    2,int16,2,Comm. count PLC,hr
 
-    template = ["address", "size", "function", "name", "description"]
-    raw_mapping = csv_mapping_parser("input.csv", template)
+    from modbus_mapper_updated import csv_mapping_parser
+    from modbus_mapper_updated import mapping_decoder
+    from pymodbus.client.sync import ModbusTcpClient
```

Comment:
I used pymodbus 3.0.0.dev4 and pymodbus.client only have this: """Initialize client."""
and i can't import modbusTcpClient

Reply:
You need to use dev, not the last release.

Reply:
Which version is dev in the pip package? 
```
pymodbus (2.5.3)
Available versions: 2.5.3, 2.5.2, 2.5.1, 2.5.0, 2.4.0, 2.3.0, 2.2.0, 2.1.0, 2.0.1, 2.0.0, 1.5.2, 1.5.1, 1.5.0, 1.4.0, 1.3.2, 1.3.1, 1.2.0, 1.1.0, 1.0.0, 0.9.0
  INSTALLED: 3.0.0.dev4
  LATEST:    2.5.3
```

Reply:
??? dev is not in pip. pip contains releases. You need to clone the repo (or work directly on GitHub), please see the README.md file.

Reply:
I'm sorry.
I tested with it, and I had changed the imports, and I had the same results than I tell you in the last updates.

Reply:
Well take a look here:
https://github.com/riptideio/pymodbus/blob/dev/pymodbus/client/__init__.py
that is the content of __init__.py on dev.

if you have a version installed with pip, depending on how you test, it might choose that over the repo version, again please see the README for suggestion on how to develop.